src/DetailsView/components/requirement-view.scss

```diff
@@ -7,3 +7,14 @@
     padding-left: 6px;
     padding-top: 1vh;
 }
+
+:global(.visual-helper) {
+    display: flex;
+    padding-top: 1.5vh;
+    &-text {
```

Comment:
I'm not sure if these styles would apply if you do not have another global wrapper around here. Can you verify?

Reply:
The most noticeable style from this is the padding-right on the text. When I inspect the visual helper and hover over visual-helper-text, it shows the padding-right being applied, as in the screenshot below:

![image](https://user-images.githubusercontent.com/48259897/81993730-3236bf00-95fb-11ea-9ce1-0637669dbcd6.png)

Reply:
I also added screenshots of the visual helper toggle to the description